wwwroot/about.html

```diff
@@ -226,86 +288,11 @@ <h3>Get started in 3 simple steps</h3>
-                                    <path d="M112.726,246.921a1.8,1.8,0,0,1,0-3.6,1.8,1.8,0,1,1,0,3.6Z" transform="translate(-89.782 -238.138)" />
-                                </svg>
-                            </a>
-                        </li>
-                        <li>
-                            <a href="https://discord.gg/nm7YHEZnJs" target="_blank">
-                                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 139 139">
-                                    <path class="cls-1" d="M86.03,63.21c-4.98,0-9.01,4.65-9.01,10.4s4.03,10.4,9.01,10.4,9.01-4.65,9.01-10.4-4.03-10.4-9.01-10.4Z" />
-                                    <path class="cls-1" d="M52.99,63.21c-4.98,0-9.01,4.65-9.01,10.4s4.03,10.4,9.01,10.4,9.01-4.65,9.01-10.4-4.03-10.4-9.01-10.4Z" />
-                                    <path class="cls-1" d="M93.31,0H45.69C20.45,0,0,20.45,0,45.69H0v47.63c0,25.23,20.45,45.68,45.68,45.69h47.64c25.23,0,45.69-20.45,45.69-45.69V45.69C139,20.46,118.55,0,93.31,0Zm25.38,95.21s-12.45,9-24.94,13.05c0,0-3.59-5.08-5.44-9,0,0,6.01-1.85,8.32-3.93l-2.08-1.85h0c-4.03,1.6-13.79,5.48-25.04,5.66-10.72,.17-19.57-3.05-25.07-5.66l-2.08,1.85c2.31,2.08,8.32,3.93,8.32,3.93-1.85,3.93-5.44,9-5.44,9-12.49-4.05-24.94-13.05-24.94-13.05-3.7-34.19,15.02-57.76,15.02-57.76,6.7-3.93,20.56-6.7,20.56-6.7l2.54,6.01c3.26-.59,7.38-.81,9.91-.88v-.04s.43,0,1.17,.01c.73-.01,1.17-.01,1.17-.01v.04c2.53,.08,6.65,.29,9.91,.88l2.54-6.01s13.86,2.77,20.56,6.7c0,0,18.71,23.56,15.02,57.76Z" />
-                                </svg>
-                            </a>
-                        </li>
-                        <li>
-                            <a href="https://t.me/WasabiWallet" target="_blank">
-                                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 139 139">
-                                    <path class="cls-1" d="M99.41,0H39.59C17.73,0,0,17.73,0,39.59v59.81c0,21.87,17.73,39.59,39.59,39.59h59.81c21.87,0,39.59-17.73,39.59-39.59V39.59C139,17.73,121.27,0,99.41,0Zm13.42,39.96l-13.75,64.19s-.83,3.79-4,3.92c-.79,.03-1.57-.2-2.25-.62-4.83-2.95-22.58-16.21-22.58-16.21l-10.62,10.13s-1.93,1.93-3.98,2.05l1.69-21.96,39.57-35.47s.72-.97,0-1.21c-.72-.24-1.13-.56-3.34,.81-2.21,1.37-48.29,30.32-48.29,30.32l-21.49-6.51s-3.36-1.28-2.87-3.38c.36-1.56,2.41-2.65,3.86-3.13,1.45-.48,82.75-31.61,82.75-31.61,0,0,3.62-1.45,5.31,1.45,1.45,2.17,0,7.22,0,7.22Z" />
-                                </svg>
-                            </a>
-                        </li>
-                    </ul>
+                    <a href="#" target="_blank" class="white-text">Wasabi Wallet</a>
                 </div>
                 <div class="col-lg py-2 text-lg-end">
                     <a href="contribution.html" class="d-inline-block me-3">Contribution</a>
                     <a href="/swagger" class="d-inline-block me-3">API</a>
```

Comment:
This should fix the broken API link in footer.

```suggestion
                    <a href="https://wasabiwallet.io/swagger/index.html" class="d-inline-block me-3">API</a>
```